specification/loadtestservice/data-plane/Microsoft.LoadTestService/stable/2022-11-01/loadtestservice.json

```diff
@@ -488,7 +488,7 @@
           "Test"
         ],
         "summary": "Associate an app component (collection of azure resources) to a test",
-        "operationId": "Test_CreateOrUpdateAppComponents",
+        "operationId": "Test_CreateOrUpdateTestAppComponents",
```

Comment:
in my opinion, as per the current SDKs this should be `Test_CreateOrUpdateTestAppComponent`